roles/haproxy/defaults/main.yml

```diff
@@ -1,5 +1,21 @@
 ---
 haproxy:
-  bufsize: 16384
   stats_group: monitor
- 
+  global:
+    maxconn: 20480
+    bufsize: 16384
+  defaults:
+    timeoute_http_request: 30s
```

Comment:
misspelling, please change all 3 occurrences in the PR to "timeout_http_request"